spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitProperties.java

```diff
@@ -399,6 +404,11 @@ public Template getTemplate() {
 		 */
 		private String trustStorePassword;
 
+		/**
+		 * Trust stroe algorithm.
+		 */
+		private String trustStoreAlgorithm;
```

Comment:
The initial value of this field should be `SunX509`. This aligns it with Spring AMQP's default for the property.